crates/re_ui/src/command.rs

```diff
@@ -95,12 +93,10 @@ impl UICommand {
 
     pub fn text_and_tooltip(self) -> (&'static str, &'static str) {
         match self {
-            #[cfg(not(target_arch = "wasm32"))]
-            Self::Save => ("Save…", "Save all data to a Rerun data file (.rrd)"),
+            Self::SaveRecording => ("Save recording…", "Save all data to a Rerun data file (.rrd)"),
 
-            #[cfg(not(target_arch = "wasm32"))]
-            Self::SaveSelection => (
-                "Save loop selection…",
+            Self::SaveRecordingSelection => (
+                "Save recording loop selection…",
```

Comment:
Hm, as a user I don't think I'd have any idea what this means... I have a suggestion but it's probably too verbose...
```suggestion
                "Save recording (current time selection only)…",